tests/installation/live_installation.pm

```diff
@@ -48,7 +48,7 @@ sub run {
         assert_and_click 'live-upgrade';
     }
     else {
-        assert_and_click 'live-installation';
+        x11_start_program('xdg-su -c "/usr/sbin/start-install.sh"', target_match => 'maximize');
```

Comment:
this is invalidating the test case as the point is to simulate what the user does without relying on internals.

Reply:
I don't agree with you. This is another way how user could do that.

Reply:
@Vogtinator This is not a problem of a product. You'll not be able to reproduce the issue manually, unless you are holding mouse click for a second or more. This is sporadic issue related to connection/vnc. When we'll revert the change we'll have the test sporadically failing and it will be false fails, as the product still works correctly.

Reply:
You're practically suggesting to replace all `assert_and_click` by `x11_start_program` here because of some weird VNC issue here, which I don't think is adequate.

The VNC issue has to be fixed in any case. That a single 0.15s press is suddenly ~7x longer is really broken.

Reply:
Please, keep in mind that test is failing for months now. We can talk about ideal world where everything works and we indirectly test something which is not in any of specs.
We tried to make it work in some way so we could address other issues one by one.
If we revert - I suggest disabling this scenario completely as it's just waste of resources.

As for replacing `assert_and_click`, we do that in scope of live installations only which we cannot make work stably.

Reply:
What about a compromise: Use this hack only on aarch64?

Reply:
+1, that's exactly what I have proposed to @OleksandrOrlov just now, as on 64bit scenario works just fine.
@OleksandrOrlov WDYT?

Reply:
This seems reasonable.

Reply:
but do you need the change at all as you already bumped RAM and CPUs for the test scenario?

Reply:
Oliver, you will not believe, but I tried to bump RAM and CPU first before introducing any new changes.

Reply:
ok, good.